gui/default/syncthing/folder/restoreVersionsModalView.html

```diff
@@ -1,4 +1,4 @@
-<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions' | translate}}" large="yes" closeable="yes">
+<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions' | translate}} {{restoreVersions.currentFolder.label}}" large="yes" closeable="yes">
```

Comment:
I think you can just use `folderLabel(id)`?

Reply:
I change that now